thanos-operator.yaml

```diff
@@ -14,6 +14,9 @@ environment:
       - go
```

Comment:
we should not be requiring these packages if we use go/build right?

Reply:
thanks for catching that, I had overlooked it. Updated the PR